src/main/scala/io/iohk/ethereum/blockchain/sync/RegularSync.scala

```diff
@@ -46,14 +51,20 @@ trait RegularSync {
 
     case PrintStatus =>
       log.info(s"Peers: ${handshakedPeers.size} (${blacklistedPeers.size} blacklisted).")
+
+    case Done =>
+      if (waitingForActor == Option(sender())) {
+        //actor is done and we did not get response
+        scheduleResume()
+      }
```

Comment:
If it's not that Actor - who could it be? Should this be logged or otherwise actioned?